pkg/tests/regress_aws_1423_test.go

```diff
@@ -27,6 +27,7 @@ import (
 )
 
 func TestRegressAws1423(t *testing.T) {
+	t.Skip("Refresh is dirty on this resource see https://github.com/pulumi/pulumi-aws/issues/3361.")
```

Comment:
This is fine. We are going to remove this test in the near future anyway. ([link](https://github.com/pulumi/pulumi-terraform-bridge/pull/1745#issuecomment-1986148079))